hr_expense_invoice/models/hr_expense_expense.py

```diff
@@ -0,0 +1,103 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+# For copyright and license notices, see __openerp__.py file in root directory
+##############################################################################
+
+from openerp import models, fields, api, exceptions, _
+
+
+class HrExpenseExpense(models.Model):
+    _inherit = 'hr.expense.expense'
+
+    @api.model
+    def line_get_convert(self, x, part, date):
+        res = super(HrExpenseExpense, self).line_get_convert(x, part, date)
+        if x.get('invoice'):
+            res['partner_id'] = (
+                x['invoice'].partner_id.commercial_partner_id.id)
+        return res
+
+    @api.model
+    def move_line_get_item(self, line):
+        if line.invoice:
+            return {
+                'account_id': line.invoice.account_id.id,
+                'name': line.name,
+                'price': line.total_amount,
+                'invoice': line.invoice,
+            }
+        else:
+            return super(HrExpenseExpense, self).move_line_get_item(line)
+
+    @api.multi
+    def action_move_create(self):
```

Comment:
Sorry, but now I don't have more time to spend on this. Can you do it for me, please?